### DISCUSSION TITLE
Get full diff of mismatching strings

### DISCUSSION THREAD

**Discussion by avin-kavish** [chaijs/chai#1666] [Support] [answered]
Is it possible to get a full diff of strings that don't match?

```
AssertionError: expected 'select "firstName" as "fullName" from…' to match /first_name/
```

Currently, it's truncated like this.

**Comment by koddsson** [comment, marked as answer]
You can set the `truncateThreshold` to whatever you want. Setting it to `0` disables truncating.

```js
chai.config.truncateThreshold = 0; // disable truncating
```

https://www.chaijs.com/guide/styles/#configuration
